packages/cli/src/commands/components/pull/actions.test.ts

```diff
@@ -142,10 +143,12 @@ describe('pull components actions', () => {
     expect(result).toEqual(mockResponse.components[0]);
   });
 
-  it('should throw an masked error for invalid token', async () => {
-    mapiClient().dispose();
+  // TODO: Ask team regarding resseting the mapi client options
```

Comment:
As a developer testing it and also as a user using it, I'd love to be able to use `createMapiClient`. Maybe we should export it?